release-team/role-handbooks/release-signal/README.md

```diff
@@ -150,7 +150,7 @@ Create an excel sheet to track various assignments during the release cycle. [Ex
 
 Roles in the assignments excel sheet:
 
-1. **Primary Reporter:** The primary reporter assigned to a task is responsible for completing it, such as providing meeting updates, giving Go/No-Go signals, etc.
```

Comment:
@Vyom-Yadav, one more doubt — during a week, what are the responsibilities of other shadows, who are not assigned either "Primary Reporter" or "Main Coordinator" role that week?

Or all shadows are assigned one/both of these roles every week?
(from the text, it sounds like the shadows who are not "Primary reporter" for a week, all of them will be working on one or other tasks of "Main Coordinator" role?
If that's the idea, let's add a line to make it clear. )

Reply:
I added the responsibilities of other shadows. Also I renamed `Primary Reporter` -> `Task Assignee` as it sounded confusing due to the word 'Primary' (based on response received from shadows). Open to better suggestions :)

Reply:
> I added the responsibilities of other shadows.

Thank you @Vyom-Yadav 

I don't have suggestions on the name of the roles (naming is always the hardest part :sweat_smile:)

Let's merge these changes, so shadows have a up-to-date handbook & if anyone have better suggestions on the name later, let's do that in a separate PR.